src/pages/settings/tabs/MetadataSitesSettings.tsx

```diff
@@ -187,7 +187,7 @@ function MetadataSitesSettings() {
       <div className="border-b border-panel-border" />
 
       <div className="flex flex-col gap-y-6">
-        <div className="flex h-[2.149rem] items-center font-semibold">TvDB Options</div>
+        <div className="flex h-[2.149rem] items-center font-semibold">TVDB Options</div>
```

Comment:
the change from TVDB to TvDB 8 did was intentional, but if you don't want it then i won't pursue it further.

Reply:
The official name has all uppercase so I kept that